run-tests.sh

```diff
@@ -0,0 +1,19 @@
+JACKD_CMD_ARGS="-r -d dummy -r 48000 -p 8192"
+
+killall -SIGKILL jackd mod-host || true
+
+# build test blocks
+pushd test-blocks
+make
+ln -s $(pwd)/build/*.lv2 ~/.lv2/
```

Comment:
pushing this to user path is not good practice, just set LV2_PATH env var here, much cleaner